benchmarks/Microsoft.AspNetCore.Routing.Performance/RouteValueDictionaryBenchmark.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public class RouteValueDictionaryBenchmark
```

Comment:
Add benchmark(s) for reading values?

Reply:
Sure, but I'm not going to put a lot of time into optimizing it right now if it's bad 😆 - my objective was to get the improvement here https://github.com/aspnet/Routing/pull/577#issuecomment-400133493 checked in